src/main/java/net/greatstart/dao/hibernate/ProjectDaoImpl.java

```diff
@@ -4,6 +4,7 @@
 import net.greatstart.model.Project;
 import org.springframework.stereotype.Repository;
 
+import javax.transaction.Transactional;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.transaction.Transactional'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)